README.md

```diff
@@ -18,3 +19,63 @@ pip install -e .
 GITHUB_TOKEN=... review-gator --config branches.yaml
 ```
 
+Basic Configuration
+-------------------
+
+Configuration is provided via a YAML file. To see an example, run 
+`review-gator --config-skeleton`
+
+```
+launchpad:
+    owners:
+        # Launchpad username
+        LAUNCHPAD-USERNAME:
+            max-age: 30
+    branches:
+        # Full path to repo
+        LAUNCHPAD-BZR-REPO-PATH:
+            review-count: 2
+github:
+    repos:
+        GITHUB-USERNAME:
+            GITHUB-PROJECT-NAME:
+                review-count: 2
+lp-git:
+    owners:
+        LAUNCHPAD-USERNAME:
+            max-age: 30
+    repos:
+        LAUNCHPAD-GIT-REPO-PATH:
+            review-count: 2
+```
+NOTE: as of v. 0.2, `owners` and all options underneath are experimental 
+with no guarantee of working. If you'd like to improve this work, we'll happily
+take PRs!
+
+
+Extras
+------------
+
+Review Gator has the ability to run tox on all your Python based codelines.
+This can be enabled on a per repo:
+
+```
+github:
+    repos:
+        CanonicalLtd:
+            review-gator:
+                review-count: 2
+                tox: true
+```
+
+To use this, tox must be installed in the environment. 
+
+Furthermore, to use all the launchpad capabilities, launchpadlib must be installed.
```

Comment:
Is "all the launchpad capabilities" or is "any launchpad capability" correct?  If it's any then I'd like to see launchpadlib by default as Canonical's code hosting solution is launchpad and it should work out of the box.  If it is only a subset of capabilities then the readme should inform the user what they are/aren't so they can choose.

Reply:
Thinking about this more, is 'tox' and dependencies large enough to push off in 'extras' rather than just making it a normal requirement?  pip installing tox in a new virtualenv pulled in: toml, py, filelock, six, appdirs, distlib, virtualenv, pluggy, pyparsing, packaging, tox which totaled about 5mb

Reply:
I think this might be a question for @fginther ? I wasn't sure if not adding tox and launchpadlib were oversights or if they were separated for a reason. I went with "separated for a reason." I actually agree that they should just get pulled in normally, but I'll defer to longer maintainers.

Reply:
We chatted this morning and decided to include tox by default and a commit has been pushed for that.